code/XMLTester.xml

```diff
@@ -2095,57 +2095,93 @@
 <!--><-->
 
 <!-->CONF-DISCOVERY-PROTOCOL<-->
-<participant profile_name="participant_discovery_protocol">
-    <rtps>
-        <builtin>
-            <discovery_config>
-                <discoveryProtocol>SIMPLE</discoveryProtocol>
-            </discovery_config>
-        </builtin>
-    </rtps>
-</participant>
+<!--
+<?xml version="1.0" encoding="UTF-8" ?>
+<dds>
+    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+-->
+        <participant profile_name="participant_discovery_protocol">
+            <rtps>
+                <builtin>
+                    <discovery_config>
+                        <discoveryProtocol>SIMPLE</discoveryProtocol>
+                    </discovery_config>
+                </builtin>
+            </rtps>
+        </participant>
+<!--
+    </profiles>
+</dds>
+-->
 <!--><-->
 
 <!-->CONF-DISCOVERY-IGNORE-FLAGS<-->
-<participant profile_name="participant_discovery_ignore_flags">
-    <rtps>
-        <builtin>
-            <discovery_config>
-                <ignoreParticipantFlags>FILTER_DIFFERENT_PROCESS | FILTER_SAME_PROCESS</ignoreParticipantFlags>
-            </discovery_config>
-        </builtin>
-    </rtps>
-</participant>
+<!--
+<?xml version="1.0" encoding="UTF-8" ?>
+<dds>
```

Comment:
According to our schema, the namespace should be declared in the root xml element

```suggestion
<dds xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
```